sdk/cmake/gcc.cmake

```diff
@@ -90,11 +90,7 @@ if(NOT CMAKE_BUILD_TYPE STREQUAL "Release")
 endif()
 
 # Tuning
-if(ARCH STREQUAL "i386")
-    add_compile_options(-march=${OARCH} -mtune=${TUNE})
-else()
-    add_compile_options(-march=${OARCH})
-endif()
+add_compile_options(-march=${OARCH} -mtune=${TUNE})
```

Comment:
Is it going to cry about the empty `-mtune=`?

Reply:
I've defined `TUNE` for all platforms